cli/src/commands/setall.js

```diff
@@ -0,0 +1,83 @@
+const fs = require('fs');
+const path = require('path');
+const minimist = require('minimist');
+const { parseYAML } = require('../format');
+
+const help = (ui, code = 0) => {
+    const { bold } = ui.colors;
+
+    ui.print(`
+${bold('NAME')}
+        comfy setall - Replace the configuration for a given environment
+
+${bold('SYNOPSIS')}
+        ${bold('comfy')} setall <environment> <path> [<options>]
+
+${bold('OPTIONS')}
+        <environment>     Name of the environment (must already exist in project)
+        <path>            Path to a configuration file (accepts json and yml formats)
+        -t, --tag=<tag>   Set a tag for this config version (default: stable)
+        -h, --help        Show this very help message
+
+${bold('EXAMPLES')}
+        comfy setall development config/comfy.json
+        comfy setall production config/api.yml -t next
+`);
+    ui.exit(code);
+};
+
+
+module.exports = (ui, modules) => function* (rawOptions) {
+    const { red, green, bold } = ui.colors;
+    const options = minimist(rawOptions);
+    const env = options._[0];
+    const configPath = options._[1];
+    const tag = options.tag || options.t || 'stable';
+
+    if (options.help || options.h || options._.includes('help')) {
+        help(ui);
+        return ui.exit(0);
```

Comment:
There, I removed it. That way, the control statements are all located in the same (main) function.